test/expected/bgw_launcher.out

```diff
@@ -435,14 +435,23 @@ SELECT wait_worker_counts(1,1,0,0);
  t
 (1 row)
 
--- clean up additional database
-\c :TEST_DBNAME :ROLE_SUPERUSER
-DROP DATABASE :TEST_DBNAME_2;
 -- test rename database
 CREATE DATABASE db_rename_test;
 \c db_rename_test :ROLE_SUPERUSER
 SET client_min_messages=error;
 CREATE EXTENSION timescaledb;
-\c :TEST_DBNAME :ROLE_SUPERUSER
+\c :TEST_DBNAME_2 :ROLE_SUPERUSER
 ALTER DATABASE db_rename_test RENAME TO db_rename_test2;
 DROP DATABASE db_rename_test2;
+-- test create database with timescaledb database as template
+CREATE DATABASE db_from_template WITH TEMPLATE :TEST_DBNAME;
```

Comment:
I would like to see testing of the fact that bgw workers get started for the new database (and also below after the alter)